client/packages/system/src/Immunisation/ListView/AppBarButtons.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {
+  DownloadIcon,
+  AppBarButtonsPortal,
+  Grid,
+  useTranslation,
+  LoadingButton,
+  EnvUtils,
+  Platform,
+} from '@openmsupply-client/common';
+
+interface ImmunisationsAppBarButtonsProps {
+  onCreate: () => void;
+}
+
+export const AppBarButtons = ({
+  onCreate,
+}: ImmunisationsAppBarButtonsProps) => {
+  const t = useTranslation('catalogue');
+
+  return (
+    <AppBarButtonsPortal>
+      <Grid container gap={1}>
+        <LoadingButton
```

Comment:
```suggestion
        <ButtonWithIcon
```

I don't know if it matters much but I don't think this needs to be be a loading button...